superset/models/core.py

```diff
@@ -583,24 +590,18 @@ def get_raw_connection(
             nullpool=nullpool,
             source=source,
         ) as engine:
-            try:
-                with closing(engine.raw_connection()) as conn:
-                    # pre-session queries are used to set the selected schema and, in the  # noqa: E501
-                    # future, the selected catalog
-                    for prequery in self.db_engine_spec.get_prequeries(
-                        database=self,
-                        catalog=catalog,
-                        schema=schema,
-                    ):
-                        cursor = conn.cursor()
-                        cursor.execute(prequery)
-
-                    yield conn
+            with closing(engine.raw_connection()) as conn:
+                # pre-session queries are used to set the selected schema and, in the
+                # future, the selected catalog
```

Comment:
I think it's finally the future and we can remove the `, in the future, the selected catalog` part?

Reply:
Ah, good point!